coriolis/api/v1/replicas.py

```diff
@@ -171,12 +176,31 @@ def _update_storage_mappings(original_storage_mappings,
 
         return storage_mappings
 
+    @staticmethod
+    def _update_user_scripts(original_user_scripts, new_user_scripts):
+        global_scripts = original_user_scripts.get('global', {})
+        new_global_scripts = new_user_scripts.get('global', {})
+        if new_global_scripts:
+            global_scripts.update(new_global_scripts)
+
+        instance_scripts = original_user_scripts.get('instances', {})
```

Comment:
We should ideally provide a way to "unset" a certain user script.
By convention, let's say that is someone posts a `null` or an empty string, that script should be unset.

Reply:
It already does that